api/protobuf-spec/github_com_openshift_origin_pkg_build_apis_build_v1.proto

```diff
@@ -851,6 +851,12 @@ message SecretBuildSource {
   optional string destinationDir = 2;
 }
 
+// SecretLocalReference contains information that points to the local secret being used
+message SecretLocalReference {
+  // Name is the name of resource being referenced
```

Comment:
"is the name of the resource"

Reply:
you don't need to comment on generated files.  just comment on the source.